rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -567,13 +588,25 @@ static s32 savedata_check_args(u32 operation, u32 version, vm::cptr<char> dirNam
 		}
 	}
 
+	if (operation >= SAVEDATA_OP_LIST_IMPORT && operation <= SAVEDATA_OP_FIXED_EXPORT)
+	{
+		if (!funcDone || userId > CELL_SYSUTIL_USERID_MAX)
+		{
+			// ****** sysutil savedata parameter error : 137 ******
+			return 137;
```

Comment:
A proof of what I said can also be found here, notice that this number is much higher than the surroundings. Usually the error numbers are placed increasingly as the code flow progrrsses. Unless it is an error that has been implemented in newer firmwares only then the number is may be higher than the rest.

Reply:
This is because the functions you reversed return different numbers than the ones I did.
They are virtually doing the same thing though, so I ignored the difference in these purely informational "error values".
In the delete functions, all of these error checks are in the same order but return starting a higher value.
I didn't want to spaghetti the code for these more or less useless internal numbers though.

Reply:
They are all the same function in assembly. Although anything that included a range check could have been handled wrongly yes because of the missing op values.

Reply:
I don't think so.
One is in lbsysutil_savedata and one is in libsysutil iirc

Reply:
Ah, alright then.